sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -24,6 +26,64 @@ def create_endpoints(plugin):
     plugin.on_message('g#', github_repo_link)
     plugin.on_message('^inspect', inspect, flags=re.IGNORECASE, mention=True, admin=True)
     plugin.on_message('^help', help_message, flags=re.IGNORECASE, mention=True)
+    plugin.on_message(STOCK_REGEX, stock_quote)
```

Comment:
For the github link integration we use the `#` special character. I think it would be best to use the same, something like `stock#<stock_name>` ?
It would have the added benefit to keep the `on_message()` regex easily understandable when we want to get a quick overview of all the triggers. You can then do, in the handler, a more advanced regex match or string manipulation